### PR TITLE
Fix rubocop warnings

### PR DESCRIPTION
**Pull request by allenwq**

### CODE REVIEW

File: app/models/concerns/course/search_concern.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ module ClassMethods
     # @param [String] keywords The keywords for filtering courses.
     # @return [Array<Course>] The courses which match the keyword. All courses will be returned if
     #   keyword is blank.
-    def search(keyword)
+    def search(keyword) # rubocop:disable Metrics/AbcSize
```
Comment (lowjoel) [inline app/models/concerns/course/search_concern.rb]: Why disable? :(

File: app/models/course/experience_points_record.rb [hunk 2]
```diff
@@ -5,7 +5,7 @@ class Course::ExperiencePointsRecord < ActiveRecord::Base
 
   belongs_to :course_user, inverse_of: :experience_points_records
 
-  scope :active, -> { where { points_awarded != nil } }
+  scope :active, -> { where { points_awarded != nil } } # rubocop:disable Style/NonNilCheck
```
Comment (lowjoel) [inline app/models/course/experience_points_record.rb]: Don't remove this, I'll be changing squeel to use nil? when my PR is merged...

File: app/models/concerns/course/search_concern.rb (hunk 1 above)
Comment (lowjoel) [inline app/models/concerns/course/search_concern.rb]: Actually, yeah I can't find any way to make it simpler lol

Comment (allenwq) [inline app/models/concerns/course/search_concern.rb]: It's a complete scope with sql joins, I think keep this method un-splitted makes more sense ?

File: app/models/course/experience_points_record.rb (hunk 2 above)
Comment (allenwq) [inline app/models/course/experience_points_record.rb]: OK.

File: app/models/concerns/course/search_concern.rb (hunk 1 above)
Comment (lowjoel) [inline app/models/concerns/course/search_concern.rb]: yes.

**Pull request merged**